Library/Formula/imagemagick.rb

```diff
@@ -48,6 +49,8 @@ class Imagemagick < Formula
   depends_on "fftw" => :optional
   depends_on "pango" => :optional
 
+  needs :openmp
```

Comment:
Sorry missed this. But in your case, it should be `needs :openmp if build.with? "openmp"`